### DISCUSSION TITLE
Getting started documentation

### DISCUSSION THREAD

**Discussion by heldergomes** [uriyyo/fastapi-pagination#769] [Ideas]
Hi,

I am currently using this lib in my project and I am handling with an issue because I need to filter fields in the Pymongo pagination and the lib nowadays can't do that !
I'd like to develop this feature and open a pull request, but I thought difficult to understand how can I contribute with this lib, I didn't find any documentation with step-by-step how can I help with this lib

**Comment by uriyyo** [comment]
Hi @heldergomes,

I am bad at writing documentation😢

 I will try to update the documentation. I will be happy if someone can help me with it)

**Reply by heldergomes** [reply]
Could you write at least the basic about how should I set up to send a pull request, and later on I can update the documentation to get prettier lol

**Reply by uriyyo** [reply]
Yup, I will try to do it today/tomorrow.

**Reply by heldergomes** [reply]
okay, thanks 🙏

**Reply by heldergomes** [reply]
Hi @uriyyo , any return about the doc ?

**Reply by uriyyo** [reply]
Hi @heldergomes,
Please check - https://uriyyo-fastapi-pagination.netlify.app/contributing/
